### PR TITLE
Implement @ineiti comments

### PR DESCRIPTION
**Pull request by tharvik**
Based on @ineiti live feedback

### CODE REVIEW

File: web/Cargo.toml [hunk 1]
```diff
@@ -15,6 +15,7 @@ chrono = { version = "0.4", features = ["wasmbind"] }
 gloo-timers = "0.2"
 http = "0.2"
 js-sys = "0.3"
+names = "0.12"
```
Comment (ineiti) [inline web/Cargo.toml:18]: Cool - so it works now out-of-the-box. Gotta replace it in fledger, too ;)

File: web/src/pages/new_account.rs [hunk 2]
```diff
@@ -140,14 +137,48 @@ impl Component for NewAccount {
...
+                    border: solid;
+                    padding: 1em 2em;
+                }
+                @media (prefers-color-scheme: dark) {
+                    .boxes > * {
+                      background-color: var(--dark-grey);
+                    }
+                }
+
+                .boxes > :nth-child(1) {
```
Comment (ineiti) [inline web/src/pages/new_account.rs:165]: Funny - I imagined a triangle with the base on the bottom - but this is also nice.

File: web/src/pages/speedtest.rs [hunk 3]
```diff
@@ -336,33 +340,35 @@ impl Speedtest {
...
-            <MatListItem>
-                <p> { "Bitcoin's TPS: 7" } </p>
-            </MatListItem>
-            <MatListItem>
-                <p> { "Ethereum's TPS: 25" } </p>
-            </MatListItem>
-        </MatList> }
+        const FIRST_COL: &str = "text-align: end; padding: 0 1em";
+
+        html! { <table>
```
Comment (ineiti) [inline web/src/pages/speedtest.rs:345]: Does that have `border=0`? I wouldn't want to see the grid of the table.

**Review by ineiti — APPROVED** [review]

File: web/Cargo.toml (hunk 1 above)
Comment (tharvik) [reply web/Cargo.toml:18]: Indeed, very easy to use, thanks for the link :)

File: web/src/pages/speedtest.rs (hunk 3 above)
Comment (tharvik) [reply web/src/pages/speedtest.rs:345]: It doesn't have it explicitly, but `border: 0px` is the default.

**Pull request merged**